examples/basics/plotting/plot_colorbars.py

```diff
@@ -0,0 +1,16 @@
+# -*- coding: utf-8 -*-
+# Copyright (c) 2015, Vispy Development Team.
+# Distributed under the (new) BSD License. See LICENSE.txt for more info.
+"""
+Show use of colorbars in plot
+"""
+import vispy.plot as vp
+
+fig = vp.Fig(size=(600, 500), show=False)
+plotwidget = fig[0, 0]
+
+plotwidget.plot([(x, x**2) for x in range(0, 100)])
+plotwidget.colorbar(position="top", cmap="viridis")
```

Comment:
argh, I get an error with `cmap=viridis` because of `IF statement nested too deeply`, but I think we already have an issue open (and maybe you started a PR?) for fixing it using texture lookups